atom/browser/api/atom_api_app.cc

```diff
@@ -380,6 +382,8 @@ mate::ObjectTemplateBuilder App::GetObjectTemplateBuilder(
       .SetMethod("setUserTasks",
                  base::Bind(&Browser::SetUserTasks, browser))
       .SetMethod("isAeroGlassEnabled", &App::IsAeroGlassEnabled)
+      .SetMethod("removeAsDefaultProtocolClient",
```

Comment:
Should we provide this API on other platforms even if it does nothing? It is a bit weird that all platforms have `set` but only one platform has `remove`.